src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -1,11 +1,22 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
 import java.util.ArrayList;
 import java.util.List;
 
+@Entity
 public class Movie implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+
     private String title;
+
+    @OneToMany
     private List<Actor> actors;
```

Comment:
Initialize the `actors` list to avoid `NullPointerException` when adding actors. You can do this by setting `private List<Actor> actors = new ArrayList<>();`.